examples/controls_example/public/app/react_control_example.tsx

```diff
@@ -207,18 +217,15 @@ export const ReactControlExample = ({
     };
   }, [controlGroupApi, timeslice$]);
 
-  if (error || (!dataViews?.[0]?.id && !loading))
-    return (
-      <EuiEmptyPrompt
-        iconType="error"
-        color="danger"
-        title={<h2>There was an error!</h2>}
-        body={<p>{error ? error.message : 'Please add at least one data view.'}</p>}
-      />
-    );
-
-  return loading ? (
+  return loadingDataView ? (
     <EuiLoadingSpinner />
+  ) : !dataView ? (
+    <EuiEmptyPrompt
```

Comment:
We should render the control group even when data view is not found. Each control must handle this error case and display errors. Instead I would prefer to keep the changes from https://github.com/elastic/kibana/pull/186582 and display warning with an explicit call to action of "installing sample web logs to run example"